src/compas_rhino/conversions/geometry.py

```diff
@@ -103,14 +103,19 @@ def point_to_compas(point):
 
     Parameters
     ----------
-    point : :rhino:`Rhino.Geometry.Point3d`
+    point : :rhino:`Rhino.Geometry.Point3d` | :rhino:`Rhino.Geometry.Point`
 
     Returns
     -------
     :class:`compas.geometry.Point`
 
     """
-    return Point(point.X, point.Y, point.Z)
+    if isinstance(point, Rhino.Geometry.Point3d):
+        return Point(point.X, point.Y, point.Z)
+    elif isinstance(point, Rhino.Geometry.Point):
+        return Point(point.Location.X, point.Location.Y, point.Location.Z)
+    else:
+        raise TypeError("Expected Rhino.Geometry.Point3d or Rhino.Geometry.Point., got: {}".format(type(point)))
```

Comment:
Instead of relying on type checking, I would rely on "feature checking". Something like this maybe?
```suggestion
    if hasattr(point, "X"):
        return Point(point.X, point.Y, point.Z)
    elif hasattr(point, "Location"):
        return Point(point.Location.X, point.Location.Y, point.Location.Z)
    else:
        raise TypeError("Unexpected point type, got: {}".format(type(point)))
```

Or maybe not even that, maybe just try..except idiom:
```python
    try:
        return Point(point.X, point.Y, point.Z)
    except MissingMemberException:
        try:
            return Point(point.Location.X, point.Location.Y, point.Location.Z)
        except MissingMemberException:
            raise TypeError("Unexpected point type, got: {}".format(type(point)))
```

Reply:
Sounds good! using try except now